build/python/build/libs.py

```diff
@@ -176,6 +177,13 @@
     patches=abspath('lib/c-ares/patches'),
 )
 
+ca = download_and_verify(
```

Comment:
This unconditionally downloads the CA file, even if you're compiling a target that doesn't use them.
Why don't you use the same thing as the Boost download does:
https://github.com/XCSoar/XCSoar/blob/812d86055bcaf69d830fa005116c5de8815f4a33/build/libboost.mk#L12-L14

Reply:
> This looks... fragile.

removed.

> This will split obscure error messages when there's no CA file in the download directory.

Changed .tgz file to be dependent upon file in d/l directory.

> Why don't you use the same thing as the Boost download does:

Made like Boost.  Note this will require kobo.mk to be updated from time to time to get the latest CA version (as with boost and many other files).

Reply:
Just nudging to see if I've satisfied all your comments.